nfsstat/tests/test_nfsstat.py

```diff
@@ -0,0 +1,79 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import mock
+import pytest
+from datadog_checks.stubs import aggregator as _aggregator
+
+from datadog_checks.nfsstat import NfsStatCheck
+
+metrics = [
+    'system.nfs.ops',
+    'system.nfs.rpc_bklog',
+    'system.nfs.read_per_op',
+    'system.nfs.read.ops',
+    'system.nfs.read_per_s',
+    'system.nfs.read.retrans',
+    'system.nfs.read.retrans.pct',
+    'system.nfs.read.rtt',
+    'system.nfs.read.exe',
+    'system.nfs.write_per_op',
+    'system.nfs.write.ops',
+    'system.nfs.write_per_s',
+    'system.nfs.write.retrans',
+    'system.nfs.write.retrans.pct',
+    'system.nfs.write.rtt',
+    'system.nfs.write.exe',
+]
+
+FIXTURE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'fixtures')
+
+
+@pytest.fixture
+def aggregator():
+    _aggregator.reset()
+    return _aggregator
+
+
+class TestNfsstat:
```

Comment:
I don't think there's any need  to use the class here. Removing it just removes a single level of indirection that we're not using here. We used it in the old one because the mocks required it but it's not necessary here. I think we wanna go for straight functions unless there's some need to do so

Reply:
I'd prefer to keep this to hold data if that's fine with you.

Reply:
I understand but I think it's just as easy to use the module to hold that data here. You're not maintaining state